x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -26,6 +26,10 @@ describe('EmptyEngineOverview', () => {
 
   it('renders a documentation link', () => {
     expect(wrapper.find(EuiButton).prop('href')).toEqual(`${docLinks.appSearchBase}/index.html`);
+
+    it('contains FlashMessages', () => {
+      expect(wrapper.find(FlashMessages)).toHaveLength(1);
+    });
```

Comment:
This is improperly nested as a result of a bad merge conflict followed by me not running the tests in this directory for awhile.  I addressed this in [60bb55f](https://github.com/elastic/kibana/pull/89816/commits/60bb55f9a751de92f0d42dd180eebc3e5585076f)